lib/msal-browser/docs/errors.md

```diff
@@ -418,3 +418,16 @@ This error occurs with MSAL.js v2.x and is due to improper configuration during
 > Your Redirect URI is eligible for the Authorization Code Flow with PKCE.
 
 ![image](https://user-images.githubusercontent.com/5307810/110390912-922fa380-801b-11eb-9e2b-d7aa88ca0687.png)
+
+### cache_quota_exceeded
+
+**Error messages**:
+
+- Exceeded cache storage capacity
+
+This error occurs when MSAL.js surpasses the allotted storage limit attempting to save token information in the [configured cache storage](./caching.md#cache-storage).
```

Comment:
Do we want to publish what storage limits are for browsers? As in 10MB for `localStorage` as an example or a link to some MDN page that may have this?

Reply:
Good point. Added a link to the MDN web page.